<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>711</width>
         <height>501</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="13" column="0">
         <widget class="QCheckBox" name="samefoldBox">
          <property name="text">
           <string>Output in Same Folder</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="extStr">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="folderStr">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="extLabel">
          <property name="text">
           <string>File extension</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="originalfnLabel">
          <property name="text">
           <string>Original Filename</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QCheckBox" name="removeBox">
          <property name="text">
           <string>Remove Original files</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="folderLabel">
          <property name="text">
           <string>Files Folder</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="outputStr">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="newfilenameStr">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="outputBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="originalfnStr">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="newfilenamLabel">
          <property name="text">
           <string>New Filename</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
          <property name="buddy">
           <cstring>newfilenameStr</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="exthintLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="folderhintLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="13" column="1" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="skipdupBox">
          <property name="text">
           <string>Skip Duplicate files</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="outputLabel">
          <property name="text">
           <string>Output Folder</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="folderBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="newnamehintLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="instruLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:green</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="outputhintLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="originalhintLabel">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>hint</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="quitBtn">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
